### ISSUE TITLE
Dead link in documentation of t.i._dumbwin32proc

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#6842]
|<img alt="tyteen4a03's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| tyteen4a03 reported|
|-|-|
|Trac ID|trac#6842|
|Type|defect|
|Created|2013-11-18 13:04:07Z|
|Branch|https://github.com/twisted/twisted/tree/broken-link-6842|

New link is http://isometri.cc/strips/gates_in_the_head.

Attachments:

* [broken-link-6842.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/c59/c5946a06ef091106692fe367f9149a06a5cb76bf/604083ad4087ae21d0fc27931b69461af080396e.patch) (386 bytes) - added by thijs on 2015-11-18 17:34:48Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__6842 6842
type__defect defect
reporter__tyteen4a03 tyteen4a03
priority__lowest lowest
milestone__ 
branch__branches_broken_link_6842 branches/broken-link-6842
branch_author__adiroiban adiroiban
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1384779847000000 1384779847000000
changetime__1447940825993368 1447940825993368
version__None None
owner__thijs thijs
cc__thijs
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_1"></a>|@thijstriemstra commented|
|-|-|

Attached patch fixes broken link.

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_2"></a>|@adiroiban commented|
|-|-|

(In [46274]) Branching to broken-link-6842.

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_3"></a>|@adiroiban set owner to @thijstriemstra|
|-|-|

Thanks for the fix...

This was also fixed in [#8025](https://github.com/twisted/twisted/issues/8025) ... but it might take some time for that patch to land so I will try to merge this.

Thanks again!

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_4"></a>|@adiroiban set status to `closed`|
|-|-|

(In [46289]) Merge broken-link-6842: Fix link in documentation of t.i._dumbwin32proc.

Author: thijs
Reviewer: adiroiban
Fixes: [#6842](https://github.com/twisted/twisted/issues/6842)
